visual-tests/test-scripts/series/stacked-bar.js

```diff
@@ -9,17 +9,38 @@
 
     d3.csv('stackedBarData.csv', function(error, data) {
 
-        var categories = data.slice(1).map(function(d) { return d.State; });
-        var yMax = d3.max(data, function(d) {
-            var total = 0;
-            for (var prop in d) {
-                if (prop !== 'State') {
-                    total += Number(d[prop]);
+        var categories = data.map(function(d) { return d.State; });
```

Comment:
Still haven't had that much experience with higher order functions yet so their use cases don't instantly jump out at me! I'll take a look at this and update it.